src/main/resources/db/changelog/changes/14-insert-project.yaml

```diff
@@ -0,0 +1,26 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert-project
```

Comment:
rename it more specific, maybe rename file to insert_sample_project because Andriy also can name that